### ISSUE TITLE
Bad icon for Download #10

### ISSUE THREAD

**Issue by j-degreef** [alexanderjeurissen/ranger_devicons#19]
Hi @alexanderjeurissen I've replied to a closed issue (Bad icon for Download #10) and I don't know if it was the right way and if you get it.

**Comment by alexanderjeurissen** [comment]
Hi @j-degreef Thanks for reporting, I have indeed noticed your comment and will take a look next weekend if I can replicate it.

**Comment by alexanderjeurissen** [comment]
@j-degreef So I took another look at your screenshot and noticed that the overlapping arrows is likely due to the ligatures that the font you're using has in place. 

Some fonts provide what's called `ligatures` which means they'll replace some character sequences with a different symbol, for example a ligature that some fonts have is replacing `>=` with a single symbol that represents smaller or equal to.

You can read more on some of the ligatures that FuraCode has by default by checking the documentation of the raw font (meaning before it was patched by NerdFont guys) https://github.com/tonsky/FiraCode

So long story short this behavior has nothing to do with ranger-devicons but is a feature of the font you selected.

I checked with the latest SauceCodePro NerdFont and the Download icon seems to work as expected, however I did notice that the Documents Icon was broken for which I'll file a different issue.

**Comment by j-degreef** [comment]
Hi,

Thanks for your explanation. I suspected it had nothing to do with ranger_devicons.
I'm been working with double-byte characters (and RTL  alignment) implementing chinese, corean, thai languages in a  english software that was not written for them so I know the problem we can have with fonts...

I can confirm that it works ok with SauceCodePro NerdFont except for the Doc icon and the GoogleDrive icon from https://fontawesome.com/cheatsheet.

What environment do you use ? It seems font rendering is not quite the same on PC and OSX. It's even not the same on VTE terminals, xterm or urxvt.
Btw, would you share the font file you use so I can test it ?

BR,
Jerome

**Comment by alexanderjeurissen** [comment]
@j-degreef Thanks for writing back, I'll quote you're comment and merge it with a more recent issue so we can have a single discussion thread about this. I'll mention you in that issue so you can participate.

https://github.com/alexanderjeurissen/ranger_devicons/issues/21

**Comment by j-degreef** [comment]
@alexanderjeurissen ok, I've seen it.
